.eslintrc

```diff
@@ -20,36 +20,175 @@
     "emotion/no-vanilla": "error",
     "emotion/styled-import": "error",
     "emotion/syntax-preference": [1, "string"],
-    "no-console": 0,
+    "no-console": 1,
     "no-underscore-dangle": 0,
-    "no-unused-vars": 1,
+    "no-unused-vars": 2,
     "no-useless-escape": 0,
-    "quotes": [1, "single", {"allowTemplateLiterals": true}],
-    "semi": [1, "always"],
+    "quotes": [2, "single", { "allowTemplateLiterals": true }],
+    "semi": [2, "always"],
+    "strict": 2,
+    "no-cond-assign": 2,
+    "no-constant-condition": 2,
+    "no-unreachable": 2,
```

Comment:
this is one I don't like as I sometimes use early returns when I am debugging, as a quick and dirty way of doing breakpoints